Automattic/wp-calypso
# Bulk domain transfer: Jetpack logo on mobile

## Before

<img width="1124" alt="image" src="https://github.com/Automattic/wp-calypso/assets/52076348/5419284a-3e59-4697-91ac-b490bdf439f7">
<img width="941" alt="image" src="https://github.com/Automattic/wp-calypso/assets/52076348/72fe90e9-cc5f-4258-9ccf-3b8378d02c6d">

## After

<img width="1148" alt="image" src="https://github.com/Automattic/wp-calypso/assets/52076348/65ee822b-ce5b-4782-9308-864c29bf0e1f">
<img width="930" alt="image" src="https://github.com/Automattic/wp-calypso/assets/52076348/e342f5e8-9453-4d76-8e69-9ed3474d3e9f">


## Testing Instructions

- Pull and run this branch or use calypso live links
- Navigate to setup/domain-transfer
- Check the jetpack logo on mobile/tablet, on landscape/portrait version

Fixes https://github.com/Automattic/wp-calypso/issues/79061
